libkmod/libkmod-config.c

```diff
@@ -135,8 +135,10 @@ static int kmod_config_add_command(struct kmod_config *config,
 	memcpy(cmd->command, command, commandlen);
 
 	l = kmod_list_append(*list, cmd);
-	if (!l)
+	if (!l) {
+		free(cmd);
```

Comment:
nops, this would be a double free. See the use of _cleanup_free_ above:

```
	_cleanup_free_ struct kmod_command *cmd;
```

Reply:
Oops, thanks for checking!